apidoc/Titanium/Filesystem/File.yml

```diff
@@ -406,7 +406,7 @@ properties:
     deprecated:
         since: "1.8.1"
         notes: Use [writable](Titanium.Filesystem.File.writable) instead.
-        removed: "6.0.0"
+        removed: "5.4.0"
```

Comment:
The removed API's are not affected, since we don't remove stuff from feature-versions, just major ones. Please revert those back to 6.0, we take of those after the 5.4.0.GA release.